bedrock/base/templates/macros.html

```diff
@@ -292,7 +292,7 @@ <h3>{{ _('Your download link was sent.') }}</h3>
 
     DO NOT include utm_source in this dictionary! (It's already a required param.)
 #}
-{% macro fxa_email_form(entrypoint, utm_source, style, class_name='fxa-email-form', utm_params={}, form_title='', intro_text='', button_text='', button_class='button button-blue') -%}
+{% macro fxa_email_form(entrypoint, utm_source, style, campaign, content, term, class_name='fxa-email-form', form_title='', intro_text='', button_text='', button_class='button button-blue') -%}
```

Comment:
Nit: suggest prefixing these new parameters with `utm_*`, so that they are consistent with the `utm_source` parameter?